src/util/get-config-files.ts

```diff
@@ -45,26 +47,25 @@ export async function getConfigFiles(
       path,
     });
 
-    return [requestedRepoFile, defaultRepoConfig];
+    files.push(defaultRepoConfig);
   }
 
   // if the configuration has no `_extends` key, we are done here.
-  if (!requestedRepoFile.config._extends) {
-    return [requestedRepoFile];
+  if (!files[files.length - 1].config || !files[files.length - 1].config._extends) {
```

Comment:
This would not work. Now `.github` entry is the last entry and the `requestedRepoFile` entry is never resolved

Reply:
This does work. Please think about it again. **If** there is a `.github` entry, then yes, it is the last entry, and yes the `requestedRepoFile` (which is the first entry) will not be resolved  **in this case because it does exist anyway**. The `.github` entry will only be added and resolved when the `requestedRepoFile` does not exist, so there is no need to resolve the first entry in that case.
So I am 99,99% sure this is correct (never say 100 ;)

Reply:
This wont be correct if both `.github` and `requestedRepoFile` and both have `extends`

Reply:
Okay no reason to use the files.push when you can simply change requestedRepoFile to defaultRepoConfig. Changing only one line :)

Reply:
> This wont be correct if both `.github` and `requestedRepoFile` and both have `extends` only `.github` would be extended.

It's impossible that both `.github` and `requestedRepoFile` both have `_extends`, only the last one in the array might have (even if it's just one element). So this is correct.